### ISSUE TITLE
Should be able to declare a rune with \u or U+

### ISSUE THREAD

**Issue by JalonSolov** [vlang/v#12391]
Right now, to declare an individual rune, you must use `` around the actual character.

It would be simpler in most cases to use something like `\uFB28` or `U+FB28` (doesn't matter) instead of hunting down the "pastable" version of `ﬨ`... especially if your editor doesn't support the full unicode range.

**Comment by JalonSolov** [comment]
Yes, you _can_ declare a rune with bare `0xFB28` (not surrounded by backticks), but it might still be more comfortable for some to use the \u or U+ syntax. I know this breaks the V "one way" philosophy, but...

**Comment by danieldaeschle** [comment]
How does Go doing it?

**Comment by JalonSolov** [comment]
Go uses single quotes and `\u`.

Here's an example
```go
package main

import (
    "fmt"
)

func main() {

    a1 := '🧺'
    a2 := '\u2665'
    a3 := '\U0001F3A8'

    fmt.Printf("%c\n", a1)
    fmt.Printf("%c\n", a2)
    fmt.Printf("%c\n", a3)
}
```

**Comment by danieldaeschle** [comment]
So in our case `\u2665` should work.

**Comment by JalonSolov** [comment]
Should... but doesn't, currently.
```
foo.v:1:13: error: invalid character literal (more than one character)
use quotes for strings, backticks for characters
    1 | a := `\u2665`
```

**Comment by JalonSolov** [comment]
`\u2665` works fine in a string... but then you've created a string, not a rune.
